### PR TITLE
Update the link

### PR DESCRIPTION
**Pull request by sanjeevkpandit**
The link to gitignore.io was misleading. Updated with the correct link.

### CODE REVIEW

**Comment by aviskarkc10** [comment]
Thanks for the 2 PRs. I merged the first one and now this one has conflicts. Could you fix it.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # gitignore
 
-A shell script to create gitignore files using [gitignore.io](gitignore.io).
+A shell script to create gitignore files using [gitignore.io](http://gitignore.io/).
```
Comment (aviskarkc10) [inline README.md]:
```suggestion
A shell script to create gitignore files using [gitignore.io](https://gitignore.io/).
```

**Review by aviskarkc10 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aviskarkc10) [reply README.md]: nice catch btw

**Comment by sanjeevkpandit** [comment]
> Thanks for the 2 PRs. I merged the first one and now this one has conflicts. Could you fix it.

Resolved.

File: README.md (hunk 1 above)
Comment (aviskarkc10) [reply README.md]: @sanjeevkpandit Why not use `https`

Comment (sanjeevkpandit) [reply README.md]: Updated.

**Review by aviskarkc10 — APPROVED** [review]

**Pull request merged**